refactor(pages): fixed promo and join us

jeydied opened the pull request:
Изменено стили у Promo и Join Us секций по макету для всех 3 - разрешений. Изменено код стайлинг для className элементов. Внесены правки в Main page в соответствии изменениями макета.

jeydied committed:
refactor(pages): fixed promo and join us

Julia-Papina commented:
Привет, некорректный нейминг классов, ранее был верный. Используем camelCase и БЭМ

jeydied committed:
refactor(pages): fixed classes naming and current projects

Исправлены: нейминги классов, размеры и отступы на странице currentProjects по макету.

jeydied committed:
fix(widgets): fix \"promo\" and \"join us\" sections

Изменены Promo и Join Us секции в соответствии с новым макетом. Исправлены нейминги классов в
соответсвии код сталийнга проекта(camelCase и БЭМ). Исправлены отступы между секциями на странице
Main Page.
